tool/site/client-react/src/components/buttonList.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import {CSSProperties, ReactNode} from "react"
+
+interface ButtonListProps {
+  children: ReactNode
+}
```

Comment:
is there anything about this component which restricts to buttons being passed in? looks more generic?

Reply:
I guess the purpose for this right now is a buttonlist, can always rename if we find other uses for it, but happy it's named as its intended use for now